.github/workflows/ci.yml

```
@@ -10,16 +10,16 @@ on:
       - release
```

```suggestion
      - release
```
Change to develop branch, release branch does not exist